### DISCUSSION TITLE
I lost access to my GitHub account

### DISCUSSION THREAD

**Discussion by ed-kurlyak** [community/community#103204] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello. I have such problem. Recently for my account was enabled 2FA. I was transferred to 2FA. I dont downloaded recovery codes. And  presently I by mistake made phone hard reset (I lost my Microsoft Authenticator for GitHub). I lost access to my GitHub account.

https://github.com/kurlyak

I created new account, and write from it now. I wrote my problem on Facebook GitHub page, and on GitHub support e-mail but nothing from them.

Is it possbile return my account to presious state before 2FA? I have my e-mail and password, I can login without 2FA. 

Is it possible recovery access to my account?

**Comment by alreadydea** [comment]
write mail in twitter and github support support@github.com hope ur account recover soon

**Comment by Rebornian48** [comment]
you can dm on twitter https://twitter.com/github or contact github support via their email on support@github.com

**Comment by ed-kurlyak** [comment]
I wrote them on https://www.facebook.com/GitHub page and on e-mail support@github.com but nothing.

**Comment by queenofcorgis** [comment]
Hi @ed-kurlyak, 

Thank you for your patience and for opening a support ticket. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
